### ISSUE TITLE
Update to electron 23

### ISSUE THREAD

**Issue by karliatto** [trezor/trezor-suite#7172]
There is a new version of electron and it is now included in nixpkgs so we probably should update.

* https://search.nixos.org/packages?channel=22.11&show=electron&from=0&size=50&sort=relevance&type=packages&query=electron
* https://www.electronjs.org/blog/electron-22-0

**Comment by karliatto** [comment]
I created new PR for including electron 23 in nix repository https://github.com/NixOS/nixpkgs/pull/216324

Once it is merged we can proceed.

**Comment by karliatto** [comment]
Latest version 23.1.1 is now available in nixpckgs https://github.com/NixOS/nixpkgs/pull/217868#event-8596479765
